README.md

```diff
@@ -0,0 +1,27 @@
+# Doppler Contact Policies API
+
+Mini-API to allow users to configure Doppler's contact policies.
+
+See [implementation details](./docs/implementation-details.md).
+
+You can test is using [VS Code REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) with [demo.http](./demo.http).
+
+## Environments
+
+### Production
+
+URL: <https://apis.fromdoppler.com/contact-policies/>
+
+The production environment is updated when a new `v1.*` tag is pushed.
+
+### QA
+
+URL: <https://apisqa.fromdoppler.net/contact-policies/>
+
+The QA environment is updated when `main` branch is updated.
+
+### INT
+
+URL: <https://apisint.fromdoppler.net/contact-policies/>
+
+The INT environment is updated when `INT` branch is updated.
```

Comment:
I known, the following information is in `demo.http` file but, what do you think?

```suggestion

## Environments

### Production

URL: <https://apis.fromdoppler.com/contact-policies/>

The production environment is updated when a new `v1.*` tag is pushed. 

### QA

URL: <https://apisqa.fromdoppler.net/contact-policies/>

The QA environment is updated when `main` branch is updated.

### INT

URL: <https://apisint.fromdoppler.net/contact-policies/>

The INT environment is updated when `INT` branch is updated.

You can check the environment´s current version using [ENVIRONMENT_URL]/version.txt
```

Reply:
Well, it is out of the original scope of this PR, for the moment I prefer keeping it as it is.